gems/quilt_rails/app/controllers/quilt/react_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Quilt
+  class ReactController < ApplicationController
```

Comment:
From @lemonmade [in the walkthrough PR](https://github.com/Shopify/cloud-docs/pull/536#discussion_r314436644):

> I am not sure about the name of this. Yes, it does render React, but is that how we want to describe it? Should this just be UiController or something along those lines?

Reply:
Yeah I agree with this 👍